<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>497</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">Jogo da Tabuada</string>
  </property>
  <widget class="QPushButton" name="btn1">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>160</y>
     <width>141</width>
     <height>111</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bitstream Charter</family>
     <pointsize>48</pointsize>
    </font>
   </property>
   <property name="text">
    <string>1</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn2">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>160</y>
     <width>141</width>
     <height>111</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bitstream Charter</family>
     <pointsize>48</pointsize>
    </font>
   </property>
   <property name="text">
    <string>1</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn3">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>160</y>
     <width>141</width>
     <height>111</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bitstream Charter</family>
     <pointsize>48</pointsize>
    </font>
   </property>
   <property name="text">
    <string>1</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblDesafio">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>90</y>
     <width>441</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>aakar</family>
     <pointsize>28</pointsize>
    </font>
   </property>
   <property name="text">
    <string>resultado</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>10</y>
     <width>58</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Jogadas:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>58</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Pontos:</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblJogadas">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>10</y>
     <width>58</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblPontos">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>30</y>
     <width>58</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblResultado">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>10</y>
     <width>361</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>aakar</family>
     <pointsize>28</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>481</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
